packages/bodiless-organisms/src/components/Menu/MegaMenu.tsx

```diff
@@ -12,34 +12,18 @@
  * limitations under the License.
  */
 
-import { flow, flowRight } from 'lodash';
+import { flow } from 'lodash';
+import { observer } from 'mobx-react-lite';
 
-import {
-  withDesign, addClassesIf, withoutProps,
-} from '@bodiless/fclasses';
+import { withDesign, addClassesIf } from '@bodiless/fclasses';
 import { withSidecarNodes, WithNodeKeyProps } from '@bodiless/core';
 import {
-  asBreadcrumb, useBreadcrumbContext, asBodilessList, asSubList,
-  withDeleteNodeOnUnwrap, asChameleonSubList,
+  asBreadcrumb, useBreadcrumbContext, asBodilessList, asChameleonSubList,
 } from '@bodiless/components';
-import { observer } from 'mobx-react-lite';
 
-import { asDefaultMenuTout, asDefaultMenuLink } from './MenuTitles';
+import { asMenuSubList } from './SimpleMenu';
+import { asDefaultMenuLink } from './MenuTitles';
 import asStylableList from '../MainMenu/asStylableList';
```

Comment:
Let's make a copy of asStylableList in the Menu directory, so we remove any dependency on the stuff in MainMenu which will be deprecated.